src/xcode/ENA/ENA/Source/Workers/Store/SecureStore.swift

```diff
@@ -262,25 +260,6 @@ final class SecureStore: SecureKeyValueStoring, Store, AntigenTestProfileStoring
 		get { kvStore["vaccinationCertificateValueDataSets"] as VaccinationValueDataSets? ?? nil }
 		set { kvStore["vaccinationCertificateValueDataSets"] = newValue }
 	}
-	
-	// MARK: - Protocol HealthCertificateValidationCaching
-	
-	var acceptanceRulesCache: ValidationRulesCache? {
```

Comment:
Do we need to have a cleanup to delete this data and the following ones?

Reply:
Good question we discussed too (also for onboardedCountries) and decided that the app can just download a fresh state for both. We do not fear a heavy load because of the staged roll-out of our app and also that not everybody requests a validation process (where onboardedCountries and dccRules are contained in).
At least, in the cache are no personalized data which would be important to keep.